client_with_ssl.js

```diff
@@ -229,21 +222,63 @@ async function selectCountriesAndCities(hzClient) {
     console.log("--------------------");
 }
 
+//
+//   This example shows how to work with Hazelcast maps, where the map is
+//   updated continuously.
+//
+//   @param client - a {@link HazelcastInstance} client.
+//
+async function nonStopMapExample(client) {
+    console.log("Now, `map` will be filled with random entries.");
+    const map = await client.getMap('map');
+
+    let iterationCounter = 0;
+    while (true) {
+        const randomKey = Math.floor(Math.random() * 100000);
+        await map.put('key' + randomKey, 'value' + randomKey);
+        await map.get('key' + Math.floor(Math.random() * 100000))
+        if (++iterationCounter === 10) {
+            iterationCounter = 0;
+            const size = await map.size();
+            console.log(`Current map size: ${size}`);
+        }
+    }
+}
 (async () => {
     try {
-        const client = await Client.newHazelcastClient(createClientConfig());
-        const map = await client.getMap('map');
+        const client = await Client.newHazelcastClient(
+            {
+                network: {
+                    hazelcastCloud: {
+                        discoveryToken: 'YOUR_CLUSTER_DISCOVERY_TOKEN'
+                    },
+                    ssl: {
+                        enabled: true,
+                        sslOptions: {
+                            ca: [fs.readFileSync(path.resolve(path.join(__dirname, 'ca.pem')))],
+                            cert: [fs.readFileSync(path.resolve(path.join(__dirname, 'cert.pem')))],
+                            key: [fs.readFileSync(path.resolve(path.join(__dirname, 'key.pem')))],
+                            passphrase: 'YOUR_SSL_PASSWORD',
+                            rejectUnauthorized: false
```

Comment:
Why do we set rejectUnauthorized ? Which option does this correspond in Java client?

Reply:
I don't think we need to set it to `false`. Can you remove it and try the sample again?